js-libs/telemetry/TelemetryV3Manager.js

```diff
@@ -174,7 +174,8 @@ TelemetryV3Manager = Class.extend({
         var eksData = {
           "type": (data && data.type) ? data.type : "workflow" ,
           "pageid": stageto,
-          "uri": (data && data.uri) ? data.uri : ""
+          "uri": (data && data.uri) ? data.uri : "",
+          "duration": Date.now() - window.PLAYER_STAGE_START_TIME || 0 
```

Comment:
Why window.PLAYER_STAGE_START_TIME is refered in Telmetry library code? 
This should not refer any events outside the telemetry library.
@pallakartheekreddy @gouravmore

Reply:
it should refer to data.duration
"duration": Date.now() - data.duration || 0